test/functional/Java8andUp/src/org/openj9/test/java/lang/ref/Test_PhantomReference.java

```diff
@@ -53,7 +53,7 @@ public void test_get() {
 	@Test
 	public void test_Constructor() {
 		ReferenceQueue rq = new ReferenceQueue();
-		bool = new Boolean(true);
+		bool = Boolean.valueOf(true);
```

Comment:
Same concern about extended life time here

Reply:
Single use.

Reply:
I've never understood what phantom does - aside from the potentially cached object, this test appears broken in two other ways:

- using sleep is extremely unreliable
- the bool variable is never set to null, so there's no guarantee it won't be collected even without the caching